src/utils.ts

```diff
@@ -25,6 +27,23 @@ export async function writeSecureFile(outputDir: string, data: string): Promise<
   return pth;
 }
 
+/**
+ * removeCachedCredentials removes any cached credentials file.
+ */
+export async function removeCachedCredentials(): Promise<void> {
+  // Look up the credentials path, if one exists. Note that we only check the
+  // environment variable set by our action, since we don't want to
+  // accidentially clean up if someone set GOOGLE_APPLICATION_CREDENTIALS or
+  // another environment variable manually.
+  const credentialsPath = process.env['GOOGLE_GHA_CREDS_PATH'];
+  if (!credentialsPath) {
+    return;
+  }
+
+  // Delete the file.
```

Comment:
Should we add some info logging after this, saying `deleted ${credentialsPath}`?